src/Card/Card.css

```diff
@@ -84,8 +84,7 @@
 }
 
 .card__title {
-    height: 98px;
-    margin-bottom: 10px;
+    margin-bottom: 30px;
```

Comment:
move visual changes to another pr, please

Reply:
maybe I also need goal, epic and ticket for each of one-line change?! 🙈🤣 please, just merge those pesky improvements and save time for both of us — there is no point to protect PR history